infrastructure/src/main/java/org/corfudb/infrastructure/DataStore.java

```diff
@@ -40,6 +47,8 @@
  */
 public class DataStore implements IDataStore {
 
+    static String EXTENSION = ".ds";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "EXTENSION" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3008)